docs/en/dev_setup/dev_env_mac.md

```diff
@@ -72,9 +55,67 @@ To setup the environment to be able to build for Pixhawk/NuttX hardware (and ins
    sudo -H python3 -m pip install --user pyserial empty toml numpy pandas jinja2 pyyaml pyros-genmsg packaging kconfiglib future jsonschema
    ```
 
-## Gazebo Classic Simulation
+## Gazebo Simulation
+
+To setup the environment for [Gazebo](../sim_gazebo_gz/index.md) simulation with Gazebo Harmonic:
+
+1. Install Gazebo Harmonic and required dependencies:
+
+   ```sh
+   brew install gz-harmonic
+   brew install qt@5 gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly gst-libav
+   ```
+
+1. Configure Qt5 and library paths by adding the following to your `~/.zshrc` file:
```

Comment:
We need to reword this. You're not configuring Qt5; we're ensuring it's accessible to the default shell, which can be anything. The majority of users will be using `~/.bashrc`; let's default to this. Any Zsh users will know what to do from here.

The homebrew Qt5 formula gives you a very verbose explanation of what needs to happen after installation. We need to make it clear this isn't specific to PX4.